backend/src/main/java/com/trackpack/app/model/notificationmodel/Notification.java

```diff
@@ -0,0 +1,18 @@
+package com.trackpack.app.model.notificationmodel;
```

Comment:
I think you need to rename all packages with 'entityType'model to just entityType. Current package path looks verbose.
.../model/notificationmodel/ -> .../model/notification/